packages/liborc/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_DESCRIPTION="Library of Optimized Inner Loops Runtime Compiler"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.4.32
+TERMUX_PKG_SRCURL=https://github.com/GStreamer/orc/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6a7349d2ab4a73476cd4de36212e8c3c6524998081aaa04cf3a891ef792dd50f
+TERMUX_PKG_BREAKS="liborc"
+TERMUX_PKG_REPLACES="liborc"
```

Comment:
Could you elaborate why package should break/replace itself? Are you sure you correctly understand what these fields do?